### ISSUE TITLE
[Boss Mechanic] Color change and floor patterns

### ISSUE THREAD

**Issue by diguliu** [AngelArenaAllstars/suggestions#168]
This is a very simple idea. 

The boss can change to different colors and each color activates "danger zones" on the floor. These "danger zones" should be fixed (therefore predictable) and each one can have specific patterns like: circles, stripes, spirals. This would force the heroes to fight the boss using special movement or tank through the damage. Each color can also have specific efects like: slow, dot, mini stun. Something like that.

**Comment by lizciz** [comment]
Nice. This mechanic can be used is a multitude of ways. One idea I though of after reading it is this:

- The whole arena floor is divided into square tiles, like a chess board.
- The tiles is one of two different colors; lets say colors A and B.
- While standing on an A tile, you get a stack of a debuff every second that does nothing. But when reaching 10 stacks, you explode dealing a lot of damage to you and any ally near by. When stepping off the tile, the debuff wears off in 10 seconds.
- While standing on a B tile, something else happens. I could not decide, so here's a few alternatives:
    - You loose 1% hp per second.
    - You loose 1 of each stat per second (or Boss steals 1 stat)
    - You are silenced and/or cannot attack
    - Your spells gain a big cooldown, such that you can only really use them once (or until you step off the tile)
    - Your character randomly slides around, such that you might slide on to an A tile again
    - You gain a stacking debuff that does damage over time.
    - You loose mana
    - You get ministunned, but for longer each time it happens
    - A combination of any of the above


Basically, B tiles are nasty and you want to avoid standing on them for too long. But A tiles are even worse, so you have to endure the B tiles a little. You have to switch tiles every now and then to reset the debuffs.

The B debuffs should wear off in about 5-7 seconds, because if you stand on an A tile for 10 seconds you explode. But the A debuff could last for 10-15 seconds, forcing you to stand on the B tiles for longer.

**Comment by prodigiousking** [comment]
No offense but the furst way seems way to easy and the second way seems too complicated and not beginner friendly

**Comment by zelding** [comment]
@prodigiousking it's good food for thought tho

**Comment by l34Um1** [comment]
Noted. Will be discussed at the next townhall meeting on the discord. Closed.
